charts/radar-self-enrolment-ui/values.yaml

```diff
@@ -226,21 +224,40 @@ networkpolicy:
         protocol: TCP
 
 # -- Set this to ORY Kratos's Admin URL
-kratosAdminUrl: "kratos-admin"
+kratosAdminUrl: "http://kratos-admin:80/admin"
 
 # -- Set this to ORY Kratos's public URL
-kratosPublicUrl: "https://localhost/kratos"
+kratosPublicUrl: "http://kratos-public:80"
 
 # -- Set this to ORY Kratos's public URL accessible from the outside world.
 kratosBrowserUrl: "https://localhost/kratos"
 
-# -- The basePath
-basePath: "/kratos-ui"
+# -- Set this to ORY Hydra's Admin URL
+hydraAdminUrl: "http://hydra-admin:4445/admin"
+
+# -- Set this to ORY Hydra's public URL
+hydraPublicUrl: "http://hydra-public:4444"
+
+# -- Set this to the REST source backend service URL
+restSourceBackendUrl: "http://radar-rest-sources-backend:8080/rest-sources/backend"
```

Comment:
do you need gateway URL as well (for non-identifiable questionnaire data?)

Reply:
Updated now. 👍